documentation/user_docs/docs/manual/Binary_operations.rst

```diff
@@ -2,63 +2,59 @@
 Binary operations
 #################
 
-Introduction
-============
+Overview
+========
 
 Binary operations between two objects can be applied in a variety of ways in
 Horace. You can either use the symbolic form (``+``, ``-`` , ``*``, ``/``,
 ``\``), or you can use the explicit function names (``plus, minus, mtimes,
-mrdivide``, ``mldivide``, etc.).
+mrdivide``, ``mldivide``).
+
+Most Horace data objects have binary operations defined between them,
+and you can perform operations between objects of different classes,
+as long as the dimensions of the "image" contained in the objects agree
+and/or the size of "pixels" array are the same.
+As described in the :ref:`FAQ <manual/FAQ:What is the difference between sqw and dnd objects>`,
+Horace data objects may contain up to two parts: an "image"
+(histogrammed counts on a grid defined by the projection axes) and a set of "pixels"
+(the raw counts from the data per detector element and energy bin).
+The ``sqw`` class has both sets of information, whereas ``d1d``, ``d2d``, ``d3d``, ``d4d``,
+``IX_dataset`` and ``sigvar`` objects contain only the "image" information,
+and ``PixelData`` objects contain only pixel information.
+
+Thus, you can perform a binary operation between an ``sqw`` object and any other Horace
+object (as long as the "image" dimensions and the number of "pixels" agree) because it
+contains both "image" and "pixel" data.
+However, you cannot perform a binary operation between a ``d1d`` and a ``PixelData`` object
+because one only has "image" data and the other only has "pixel" data.
+
+You can perform a binary operation between a scalar number and any Horace data object.
+
+You can perform a binary operation between a numeric Matlab array and any Horace data object
+which contains an "image" as long as the array dimension and image dimension matches.
+
+The result of the binary operation has the same type as the more "complex" of the input objects,
+where "complexity" equates to the amount of information an object carries.
+That is, an ``sqw`` object, which has both "image" and "pixel" data, is considered the most complex,
+whereas a ``sigvar`` object which just has count data (a signal and a variance array) but no
+coordinate information is considered the least complex.
```

Comment:
Want an admonition (`.. note`?) of a list of the "hierarchy of complexity" here?

Reply:
Hmm... I don't know exactly what the hierarchy is and was too lazy to look at the code to find out...